Vic2ToHoI4/Source/HOI4World/Regions/Regions.cpp

```diff
@@ -8,8 +8,28 @@ std::optional<std::string> HoI4::Regions::getRegion(int provinceNumber) const
 	{
 		return regionsMap.at(provinceNumber);
 	}
-	else
+
+	return std::nullopt;
+}
+
+
+std::optional<std::string> HoI4::Regions::getRegionName(const std::string& regionName) const
+{
+	if (regionNames.contains(regionName))
 	{
-		return std::nullopt;
+		return regionNames.at(regionName);
```

Comment:
You check if regionNames contains regionName in the condition, so
```suggestion
		return regionNames[regionName];
```
should be OK.
Also, replacing the block with regionNames.find(), if (!= regionNames.end()) would save one lookup.